.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Publish to npm'
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
```

Comment:
Do we actually need a matrix, if we're only publishing for one node-version?

Reply:
Same thing here, setup-node supports adding cache for yarn

Reply:
Don't know about the matrix, but a single column, single row matrix is still just one element, right? I suppose the fix would be to remove the `strategy` property and instead use this in the step:

```yml
with:
  node-version: '14'
```

But if we do want to expand the matrix for later, then I don't mind leaving it in. Also provides an easy place to look for versions, I think 💁🏼

As for the caching, yeah, I'll add it!